app/assets/stylesheets/shared/form.scss

```diff
@@ -112,6 +112,10 @@ label[required="true"] {
       border-color: $red;
     }
 
+    &.prx_dragndrop {
+      border: 2px dashed $gray-300;
+    }
```

Comment:
Adds a custom class to apply to file fields - I had to do this to overwrite the form-control border